simpl/config.py

```diff
@@ -213,6 +213,7 @@ def __init__(self, *args, **kwargs):
         self._mutexgroup = None
 
     def __copy__(self):
+        """TODO(zns): add docstring."""
```

Comment:
Is this just here to suppress PEP8 warnings? Why not just let the warnings continue until a proper docstring is added?